c2cgeoform/templates/grid.pt

```diff
@@ -32,9 +37,10 @@
                   return '_id_';
               },
               to: function(value) {
+                  var delete_url = "${request.route_url('c2cgeoform_action', action='delete', id='_id_')}".replace('_id_', value);
```

Comment:
this does not match with route c2cgeoform_delete, seems that this route is not used